content/docs/programs/immunizations.md

```diff
@@ -0,0 +1,131 @@
+
+If you need to edit the name of the Immunization Program, do so by updating the input field at the top of the detail screen.
+
+Once you are happy with your changes, use the `Save` button in the bottom right to save them.
+
+### Vaccine Courses
+
+The Immunization Program detail view presents you with a list of Vaccine Courses that are a part of the program.
+
+The list of Vaccine Courses is divided into 3 columns:
+
+| Column                 | Description                                           |
+| :--------------------- | :---------------------------------------------------- |
+| **Name**               | The name of the Vaccine Course                        |
+| **Target Demographic** | The demographic to receive the vaccine Vaccine Course |
+| **Doses**              | The number of doses included in the course            |
+
+### Adding a new Vaccine Course
+
+To add a new Vaccine Course, click the `Add vaccine course` button in the top right.
+
+![vaccine course: add button](/docs/programs/images/vaccine_course_add_button.png)
+
+This will open a new window, where you can enter the name of the new Vaccine Course.
+
+![vaccine course: create modal](/docs/programs/images/vaccine_course_add.png)
+
+- Click `OK` once you are happy with your changes
+  - You will be redirected to the detail view of your newly created Vaccine Course
+- OR: Click `Cancel` at any time to revert your changes
+
+### Deleting a Vaccine Course
+
+You can delete Vaccine Course from the list.
+
+Select the Vaccine Course that you wish to delete by checking the box on the right end of the list. You can select more than one Vaccine Course to be deleted at once. You can even select them all using the master checkbox in the table header.
+
+![vaccine courses: select to delete](/docs/programs/images/vaccine_courses_select.png)
+
+Open the `Actions` dropdown in the top right corner, above the list, and select `Delete selected lines`.
+
+![vaccine courses: actions](/docs/programs/images/immunizations_actions.png)
+
+You will be asked to confirm that you wish to delete the selected Vaccine Courses. Press `Cancel` to prevent deleting, or `Ok` to confirm.
+
+![vaccine courses: confirm delete](/docs/programs/images/vaccine_courses_confirm_delete.png)
+
+A notification in the bottom left corner will confirm how many courses were deleted.
+
+## Viewing Vaccine Course Details
+
+To view a Vaccine Course in more detail, click on it. This will take you to the detail page for that course.
+
+![vaccine course: detail](/docs/programs/images/vaccine_course_detail.png)
+
+You can use the `Close` button in the bottom right to close the detail view, and go back to the list of Vaccine Courses within this Immunization Program.
+
+From this view, you can edit all the relevant information for the Vaccine Course, including its name, details like its wastage rate, and the number of doses.
```

Comment:
Maybe its worth adding here, demographics are configured in demographics page, vaccine items are from mSupply?? wasn't sure how much this page would change before adding much detail lol